openshift/release/generate-release.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source $(dirname $0)/resolve.sh
+
+root_dir=$(dirname $0)/../..
+
+release=$(yq '.project.tag' openshift/project.yaml)
+release=${release/knative-/}
+
+echo "Release: $release"
+
+./openshift/generate.sh
+
+artifacts_dir="openshift/release/artifacts"
+rm -rf $artifacts_dir
+mkdir -p $artifacts_dir
+
+image_prefix="registry.ci.openshift.org/openshift/knative-${release}:knative-"
+tag=""
+
+plugins="${artifacts_dir}/backstage-plugins.yaml"
```

Comment:
I feel like we should call this "backstage-eventmesh-backend.yaml`

Reply:
Yes, but we don't want to bundle them all together

Reply:
Ok, fixed